Ginger/GingerCore/Environments/Database.cs

```diff
@@ -263,24 +281,26 @@ public string GetConnectionString()
                 }
                 else if (DBType == eDBTypes.DB2)
                 {
-                    connStr = "Server=" + TNSCalculated + ";Database=" + Name + ";UID=" + UserCalculated + "PWD=" + deCryptValue;
+                    //TODO: need to confirm connection string works aftr UID => ';' is missing
+                    //standard connection string format for DB2 is Server=myAddress:myPortNumber;Database=myDataBase;UID=myUsername;PWD=myPassword;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5006741)